css/helper/helper.css

```diff
@@ -0,0 +1,10 @@
+@import './variables.css';
+@import './reset.css';
+@import './background.css';
+@import './display.css';
+@import './flex.css';
+@import './position.css';
+@import './sizing.css';
+@import './spacing.css';
+@import './typography.css';
+@import './border.css';
```

Comment:
you can put them in one CSS file

Reply:
Yes I am importing all **helper classes** in a single **helper.css** file and then importing it to **main.css**. Doing similar thing for component classes also, as I thought it would be easy to maintain.
Please let me know if you meant something else with **putting them in one CSS file**

Reply:
yes i mean same thing